site/site_test.go

```diff
@@ -48,7 +48,7 @@ func TestCaching(t *testing.T) {
 	defer srv.Close()
 
 	// Create a context
-	ctx, cancelFunc := context.WithTimeout(context.Background(), 1*time.Second)
+	ctx, cancelFunc := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
(Idea, out-of-scope): should we have a "standard" set of timeouts we use for all unit tests?
e.g. "short" -> 1 second, "medium" -> 5 seconds, "long", 10 seconds?

Reply:
I really like that idea! It's all over the place now and doesn't allow us to easily adjust for slow/fast platforms.